tests/testthat/_snaps/compare-data-frame.md

```diff
@@ -0,0 +1,70 @@
+# informative diff for additions and deletions
+
+    Code
+      df <- data.frame(x = 1:5, y = 5:1)
+      compare(df, unrowname(df[1:3, ]))
+    Output
+      `attr(old, 'row.names')`: 1 2 3 4 5
+      `attr(new, 'row.names')`: 1 2 3    
+      
+      old vs new
+              x y
+        [1, ] 1 5
+        [2, ] 2 4
+        [3, ] 3 3
+      - [4, ] 4 2
+      - [5, ] 5 1
```

Comment:
This feels a bit weird. In this case, the `old$x` vs. `new$x` comparisons definitely feel easier to interpret. From the above, I feel unsure who is missing these rows (I mean, I know, but it's not super obvious from what I'm looking at).

Reply:
How about this?

```
old vs new
           x y
  old[1, ] 1 5
  old[2, ] 2 4
  old[3, ] 3 3
- old[4, ] 4 2
- old[5, ] 5 1
```

Reply:
That feels worse 😂 because now I'm pretty sure that `old` is missing those rows (except the truth is that they're missing from `new`).

Reply:
It's telling you the changes from `old` you need to make to get `new`; that at least is consistent across all waldo output.